### ISSUE TITLE
mbedtls ssl handshake error

### ISSUE THREAD

**Issue by cryptrr** [Javernaut/ffmpeg-android-maker#98]
When you open a direct link with ffmpeg built with mbedtls

```
[tls @ 0xb4000078345a2430] mbedtls_ssl_handshake returned -0x6c00

[in#0 @ 0xb4000078045a4010] Error opening input: I/O error
```

**Comment by Javernaut** [comment]
According to [this](https://github.com/Mbed-TLS/mbedtls/issues/9545#issuecomment-2333337755), the issue was fixed by mbedtls 3.6.1 (currently the project uses 3.6.0).

Try it, and give a feedback please.

**Comment by cryptrr** [comment]
Updating the version to 3.6.1 fixes this. Confirmed.
